CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [2.6.0](https://github.com/mixxxdj/mixxx/milestone/44) (Unreleased)
 
+### SETM support
+
+* Add simple support for STEM files [#12744](https://github.com/mixxxdj/mixxx/pull/12744)
```

Comment:
Looks like the wrong issue. Did you mean to point to #13044 ?

Reply:
Thank you for noticing. Fixed!